src/CSET/recipes/extract_instant_model_level_air_temp.yaml

```
@@ -0,0 +1,36 @@
+title: Extract Meaned Model level Air Temperature
+description: |
+  Extracts out the instantaneous model level air temperature from a file and writes it
+  to a new one.
+section: Grid Stat
+major_cat: Major Category 1
+minor_cat: Minor Category 1
+
+steps:
+  - operator: read.read_cubes
+    constraint:
+      operator: constraints.generate_var_constraint
+      varname: air_temperature
+
+  - operator: filters.filter_cubes
+    constraint:
+      operator: constraints.combine_constraints
+      varname_constraint:
+        operator: constraints.generate_var_constraint
+        varname: air_temperature
+      model_level_number_constraint:
+        operator: constraints.generate_model_level_constraint
+        model_level_name: 2
```

```suggestion
        model_level_number: 2
```
Strictly speaking this passes in the field name as a number rather than a string. You would have to quote it (`"2"`) for it to be a string. I've suggested some addition logic to the operator so this doesn't matter.